server/app/mutations/stacks/create.rb

```diff
@@ -47,6 +61,26 @@ def execute
         return
       end
 
+      if self.volumes
+        # TODO Separate into own method
+        attributes.delete(:volumes)
+        self.volumes.each do |volume|
+          if volume[:external]
+            vol = Volume.where(name: volume['name'], grid: grid, stack: nil).first
+            if vol
+              stack.external_volumes.create!(volume: vol)
```

Comment:
Hmm... you don't set any `name: ...`? But the  `external_volume.name` is always the same as the `external_volume.volume.name`?